src/aliceVision/calibration/checkerDetector.cpp

```diff
@@ -656,48 +665,69 @@ bool CheckerDetector::getSeedCheckerboard(Eigen::Matrix<IndexT, -1, -1> & board,
 }
 
 
-bool CheckerDetector::getCandidates(std::vector<NewPoint> & candidates, Eigen::Matrix<IndexT, -1, -1> & board)
+bool CheckerDetector::getCandidates(std::vector<NewPoint> & candidates, const CheckerBoard & board, bool inside) const
 {   
     if (board.rows() < 2)
     {
         return false;
     }
 
-    
-    for (int col = 0; col < board.cols(); col++)
+    if (!inside)
     {
-        NewPoint p;
-        p.col = col;
-        p.row = -1;
-        p.score = std::numeric_limits<double>::max();
-
-        for (int row = 0; row < board.rows() - 2; row++)
+        for (int col = 0; col < board.cols(); col++)
```

Comment:
```suggestion
        for (int col = 0; col < board.cols(); ++col)
```
make `col` coherent with the type returned by `board.cols()`